src/utils.coffee

```diff
@@ -44,13 +44,18 @@ class Utils
     result = [result] unless result instanceof Array
     _.compact(result)
 
-  @wrapObjects: (array) ->
+  @wrapObjects: (array, options = {}) ->
     output = []
     _(array).each (elem) =>
       if elem.constructor == Object
         for key, value of elem
           o = {}
-          o[key] = @wrapObjects(if value instanceof Array then value else [value])
+
+          if options.ignoreCollections && elem[key] instanceof Backbone.Collection
```

Comment:
@juanca Passing the option makes it such that if wrapObjects encounters a Backbone.Collection, it won't wrap it within an object. Where should we document something like this?

Would you prefer to see something like `wrapObjectsForIncludes` as a separate function with this behavior and no options? We considered going down that path but felt in that case, `wrapObjectsForIncludes` and `wrapObjects` would share so much in common. Thoughts?